corehq/apps/case_search/tests/test_filter_dsl.py

```diff
@@ -344,6 +344,165 @@ def test_nested_filter(self):
         built_filter = build_filter_from_ast("domain", parsed)
         self.checkQuery(expected_filter, built_filter, is_raw_query=True)
 
+    def test_selected(self):
+        parsed = parse_xpath("selected(first_name, 'Jon')")
+        expected_filter_single = {
```

Comment:
This query is incorrect. `selected` should match if the property contains the term:

```
first_name = 'Jon James'   # match
first_name = 'Mick Jon'   # match
first_name = 'Mick Jo'   # no match (unless also using fuzzy)
first_name = 'Mick'   # no match
```

Reply:
Then in that case the query for correct `selected` should be identical to `selected-any` and look like this?

```
"must": {
  "match": {
    "case_properties.value": {
       "query": "Jon",
       "operator": "or",
       "fuzziness": "0"
     }
  }
}
```

Regardless of how many terms we're searching on?

Reply:
that's correct

Reply:
👍  corresponding changes made with recent commits